docs/index.rst

```diff
@@ -73,7 +73,10 @@ Instrumentation
 Web
 ~~~
 
-We support many :ref:`web-frameworks`. Install the middleware for yours.
+We support many `web frameworks`. Install the middleware for yours.
```

Comment:
Thanks for spotting that! the underscore is missing so it should be `web frameworks_` otherwise it will be written in _italic_.